app/posts/register-trainee-teachers/2021-08-03-homepage-update-july-21.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Homepage update July 21
+description:
+date: 2021-08-03
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Changes page heading from the service name (‘Register trainee teachers’) to ’Your trainee teachers’, to:
+
+- reduce confusion with the start page that has the same heading
+- make the purpose of the page a little clearer
+
+Adds the organisation name to the heading, this sets context for the ‘Your’ in ‘Your trainee teachers’ and will for users that belong to more than one organisation.
+
+Makes the heading one size larger for consistency with the rest of the service and gives room in heading hierarchy for sections in the page.
+
+Updates the nav to refer to ‘trainees’ consistently instead of ‘records’. The service currently uses ‘records’ and ‘trainees’ either inconsistently or in a way thats pretty subtle — I personally find ‘record’ to be too general and find it helpful to use ‘trainee’.
```

Comment:
```suggestion
Updates the nav to refer to ‘trainees’ consistently instead of ‘records’. The service currently uses ‘records’ and ‘trainees’ either inconsistently or in a way that’s pretty subtle — I personally find ‘record’ to be too general and find it helpful to use ‘trainee’.
```

Reply:
Not sure we should refer to ourselves in first person. Can we frame this as a research question?